setup/install_gatling/00_pre_install/20_rpm_setup.rb

```diff
@@ -33,11 +33,6 @@
     on hosts, 'yum install -y lsof || true'
   end
 
-  step 'enable atop service' do
-    on hosts, 'chkconfig atop on'
-    on hosts, 'service atop start'
-  end
```

Comment:
this isn't required now?

Reply:
No, it's being managed by beaker-benchmark now.